README.md

```diff
@@ -1,4 +1,56 @@
 # Bootstrap Playground
 
-## Getting started
+## Host the Webpage locally
+Inorder to see the webpage working and with all function skip the *Getting started* part and go to the [*local Hosting*](#local-hosting) part and download this [empty](https://github.com/stetsche/bootstrap-playground) or this [sample repository](https://github.com/SirSaibot/bootstrap-playground) and use it instead of forking your own repository. 
 
+## Getting started <a name="gettingstarted"></a>
+### Installing needed programms
+
+First of you need to install some programms incase you don't already have them:
+- NodeJS (https://nodejs.org/en/download)
+- Github Desktop (https://desktop.github.com) or Git (https://git-scm.com/downloads)
+
+Other optional programms I recommend:
+- VisualStudio Code (https://code.visualstudio.com/download)
+- a nice Browser (not Internet Explorer or Microsoft Edge)
+
+After downloading and installing everything, you want to go to the [boostrap-playground repository](https://github.com/stetsche/bootstrap-playground) (https://github.com/stetsche/bootstrap-playground) and in the top-right click on the Fork button.
+![image of button](readme-img/Fork.png)
+
+On the page that opens you can change the owner, name and description but it's recommended to leave everything as it is and press *create fork*.
+After a few seconds it should be done, now you can look at your own repositories and there should be a new one named *bootstrap-tutorial* (if you didn't change the name).
+
+This tutorial is for GithubDesktop only. It is very much possible archive the same with Git or similar options but this guide focuses on GithubDesktop.
+
+### GithubDesktop
+Once you have opened GithubDesktop, there should be the option to open one of your repositories. Choose the just forked repository. After a couple of seconds everything should be ready. 
+
+If you did not install Visual Studio you can skip the next step and open the files index.html and style.css manually in other programms like notepad.
+
+### Visual Studio
+Open VisualStudio and on the top left click on *Explorer*. There should be the option to open a folder. Click on that and the navigate towards the local path of the just created repository. Afterwards you should see some files and folders coming up on the left side. Open index.html and style.css by simply clicking on them. 
```

Comment:
Filenames and paths can be `nicely formatted` by wrapping them in this quotes: \`index.html\`